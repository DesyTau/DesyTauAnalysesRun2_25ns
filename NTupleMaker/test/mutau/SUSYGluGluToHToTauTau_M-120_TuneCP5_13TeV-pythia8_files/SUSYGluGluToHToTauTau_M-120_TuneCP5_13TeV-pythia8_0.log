000 (2509993.000.000) 01/17 11:47:05 Job submitted from host: <131.169.223.40:9618?addrs=131.169.223.40-9618+[2001-638-700-10df--1-28]-9618&noUDP&sock=7812_f5f0_32>
...
001 (2509993.000.000) 01/17 11:48:35 Job executing on host: <131.169.84.202:44748?addrs=131.169.84.202-44748>
...
006 (2509993.000.000) 01/17 11:48:44 Image size of job updated: 117556
	41  -  MemoryUsage of job (MB)
	41300  -  ResidentSetSize of job (KB)
...
005 (2509993.000.000) 01/17 11:49:02 Job terminated.
	(1) Normal termination (return value 255)
		Usr 0 00:00:16, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:16, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	0  -  Run Bytes Sent By Job
	0  -  Run Bytes Received By Job
	0  -  Total Bytes Sent By Job
	0  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     0.01        1         1 
	   Disk (KB)            :     1     3072000   3244370 
	   Memory (MB)          :    41        2048      2048 
...
