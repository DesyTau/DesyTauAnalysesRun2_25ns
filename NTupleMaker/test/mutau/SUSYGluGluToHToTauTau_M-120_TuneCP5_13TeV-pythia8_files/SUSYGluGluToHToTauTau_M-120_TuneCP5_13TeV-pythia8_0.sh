source /cvmfs/cms.cern.ch/cmsset_default.sh
export SCRAM_ARCH=slc6_amd64_gcc630
cd /nfs/dust/cms/user/cardinia/HtoTauTau/HiggsCP/CMSSW_9_4_9/src
cmsenv
cd -
SynchNTupleProducer_2017 analysisMacroSynch_lept_mt_SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8.conf SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8_0 mt
