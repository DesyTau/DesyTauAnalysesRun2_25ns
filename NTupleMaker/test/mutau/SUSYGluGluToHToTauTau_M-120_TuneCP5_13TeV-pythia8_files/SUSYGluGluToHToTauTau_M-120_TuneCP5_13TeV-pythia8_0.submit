+RequestRuntime=10000

RequestMemory = 2000

executable = SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8_0.sh

transfer_executable = True
universe            = vanilla
getenv              = True
Requirements        = OpSysAndVer == "SL6"

output              = SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8_0.out
error               = SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8_0.error
log                 = SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8_0.log

queue

