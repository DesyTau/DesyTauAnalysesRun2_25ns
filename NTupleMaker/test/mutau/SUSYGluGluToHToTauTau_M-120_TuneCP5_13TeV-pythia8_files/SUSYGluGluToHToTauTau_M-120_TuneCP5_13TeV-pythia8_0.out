/nfs/dust/cms/user/cardinia/HtoTauTau/HiggsCP/CMSSW_9_4_9/src/DesyTauAnalyses/NTupleMaker/test/mutau/SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8_files
Number of single lepton trigger legs = 2
Number of X trigger legs (lep leg)   = 2
Number of X trigger legs (tau leg)   = 2
Number of ditau trigger legs         = 0
dxyLeptonCut 0.045
dzLeptonCut 0.2
/pnfs/desy.de/cms/tier2/store/user/mmeyer/ntuples/MCFall17/Signals//SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8/SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8_0.root
/pnfs/desy.de/cms/tier2/store/user/mmeyer/ntuples/MCFall17/Signals//SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8/SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8_1.root
/pnfs/desy.de/cms/tier2/store/user/mmeyer/ntuples/MCFall17/Signals//SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8/SUSYGluGluToHToTauTau_M-120_TuneCP5_13TeV-pythia8_2.root
Histogram GluGluHToTauTau_M125_13TeV_powheg_pythia8_pileup is not present in pileup file
